modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -32,7 +33,139 @@ With those steps completed, stop your webserver.
 sudo service apache2 stop
 ----
 
-[[backup-your-existing-installation]]
+== Backup Your Existing Installation
+
+First, xref:maintenance/backup.adoc[backup] the following items:
+
+.  The ownCloud server data directory
+.  The complete ownCloud directory
+.  The ownCloud server database
+
+You can find detailed steps to do backups here: xref:maintenance/backup.adoc[].
+
+[source,console]
+----
+
+# This example assumes Ubuntu Linux and MariaDB
+mv /var/www/owncloud /var/www/owncloud-old-version-number
+mysqldump -u<username> -p<password> <databasename> > <ownCloud-Version-Dump.sql>
+----
+
+== Review Third-Party Apps
+
+# Enable maintenance mode using the occ command.
```

Comment:
If I follow the document I am already in maintenance mode, right?

Reply:
fixed